pom.xml

```diff
@@ -672,6 +630,48 @@
           </execution>
         </executions>
       </plugin>
+            <plugin>
```

Comment:
Indentation off here.

Also - did the order of these need to change? Makes the diff larger and harder to read.